simplq/src/components/pages/Admin/index.jsx

```diff
@@ -14,8 +14,8 @@ import SidePanel from './AdminSidePanel';
 import StandardButton from '../../common/Button';
 import Ribbon from '../../common/Ribbon';
 import QRCode from '../../common/Popup/QrCode';
-import { disableScroll, enableScroll } from './ControlScroll';
-import { getToursteps, hasUserBeenOnTour } from './TourSteps';
+import { disableScroll, enableScroll } from "./ControlScroll";
+import { getToursteps, hasUserBeenOnTour, stepChange } from "./TourSteps.jsx";
 import useRequest from '../../../api/useRequest';
```

Comment:
Could you remove the .jsx here? It's not requried.

Reply:
Done